packages/helper-settings/index.js

```diff
@@ -9,35 +9,13 @@ const defaults = {
     since: Date.now(),
     counter: 0,
   },
-  enableBetterPHP: false,
 };
 
-function isTestEnv() {
-  if (process.env.NODE_ENV === 'test') {
-    return true;
-  }
-
-  if (
-    typeof window !== 'undefined' &&
-    /OctoLinker\/OctoLinker\/blob\/.+\/e2e\/fixtures/.test(
-      window.location.pathname,
-    )
-  ) {
-    return true;
-  }
-
-  return false;
-}
```

Comment:
I wasn't sure if this should be left since it's not used anymore.

Reply:
I think that's fine. I thought about using feature flags for other things as well, but given that testing isn't that straight forward we shouldn't continue with this approach